### PR TITLE
[Samples] Add 17.multilingual-bot sample

### LINKED ISSUE #868: Sample: 17.multilingual-bot

### PR DESCRIPTION
**Pull request by Batta32**
Fixes #868

- [X] Migration
- [X] Testing

## Description

We are adding the sample 17.multilingual-bot in Java having into account the [C# 17.multilingual-bot](https://github.com/microsoft/BotBuilder-Samples/tree/main/samples/csharp_dotnetcore/17.multilingual-bot) present in [BotBuilder-Samples](https://github.com/microsoft/BotBuilder-Samples) repository.

## Specific Changes

  - Generate main structure of the sample
  - Add `bots` folder with the `MultiLingualBot.java` class
  - Add `cards` folder with `welcomeCard.json`
  - Add `deploymentTemplates` folder
  - Add `translation` folder including the `model` folder and classes
  - Add root files
  - Add `application.properties`
  - Add LICENSE document
  - Add README document to describe the steps to run the sample

## Testing

We are currently testing the migrated sample. As soon as it's validated, it will be ready to be merged.

_Current behavior of the Java migrated sample_
image

### CODE REVIEW

File: samples/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/MultiLingualBot.java [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+                {
+                    setActions(actions);
+                }
+            };
+            reply.setSuggestedActions(suggestedActions);
+            return turnContext.sendActivity(reply).thenApply(resourceResponse -> null);
+        }
+    }
+
+    private static CompletableFuture<Void> sendWelcomeMessage(TurnContext turnContext) {
```
Comment (tracyboehrer) [inline samples/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/MultiLingualBot.java:130]: sendWelcomeMessage will only send to the first non-target member.  This probably works fine when there is only one.

For multiples, need to build a list of CompletableFuture.

Something like:

```    
private static CompletableFuture<Void> sendWelcomeMessage(TurnContext turnContext) {
        return turnContext.getActivity().getMembersAdded().stream()
            .filter(
                member -> !StringUtils
                    .equals(member.getId(), turnContext.getActivity().getRecipient().getId())
            )
            .map(channel -> {
                Attachment welcomeCard = MultiLingualBot.createAdaptiveCardAttachment();
                Activity response = MessageFactory.attachment(welcomeCard);
                return turnContext.sendActivity(response)
                    .thenCompose(task -> turnContext.sendActivity(MessageFactory.text(WELCOME_TEXT)));
            })
            .collect(CompletableFutures.toFutureList())
            .thenApply(resourceResponses -> null);
}
```

**Review by tracyboehrer — CHANGES_REQUESTED** [review]

File: samples/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/MultiLingualBot.java (hunk 1 above)
Comment (Batta32) [reply samples/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/MultiLingualBot.java:130]: Thanks @tracyboehrer, we will be reviewing the requested change and we will update the PR 😊.

Comment (Batta32) [reply samples/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/MultiLingualBot.java:131]: @tracyboehrer - we already applied the requested changes in this [commit](https://github.com/microsoft/botbuilder-java/pull/898/commits/88293caf4e959199380177b52f9e6b7c3be499c5).

We are currently validating the functionality, as soon as we finish with the validation we will update the PR to be out of draft.

**Review by tracyboehrer — APPROVED** [review]

**Review by LeeParrishMSFT — APPROVED** [review]

**Comment by Batta32** [comment]
@tracyboehrer & @LeeParrishMSFT - the sample is correctly working and the PR is **ready to be merged**!

**Pull request merged**